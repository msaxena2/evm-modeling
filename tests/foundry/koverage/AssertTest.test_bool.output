{
  true
#Equals
  notBool ( 0 <=Int CALLER_ID andBool ( 0 <=Int ORIGIN_ID andBool ( 0 <=Int NUMBER_CELL andBool ( CALLER_ID <Int pow160 andBool ( ORIGIN_ID <Int pow160 andBool ( NUMBER_CELL <=Int maxSInt256 andBool ( VV0_x_114b9705 ==Int 0 ==Bool true ==Bool true ) ) ) ) ) ) )
}

{
  false
#Equals
  0 <=Int CALLER_ID andBool ( 0 <=Int ORIGIN_ID andBool ( 0 <=Int NUMBER_CELL andBool ( CALLER_ID <Int pow160 andBool ( ORIGIN_ID <Int pow160 andBool ( NUMBER_CELL <=Int maxSInt256 andBool ( VV0_x_114b9705 ==Int 0 ==Bool true ==Bool true ) ) ) ) ) )
}

