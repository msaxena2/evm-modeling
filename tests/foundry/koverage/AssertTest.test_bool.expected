{
  false
#Equals
  VV0_x_114b9705 ==Int 0 ==Bool true ==Bool true
}

